CryptoLightNews/Networking/Endpoints.swift

```diff
@@ -0,0 +1,32 @@
+//
+//  Endpoints.swift
+//  CryptoLightNews
+//
+//  Created by Richard Price on 21/12/2022.
+//
+
+import Foundation
+
+protocol APIBuilder {
+    var urlRequest: URLRequest { get }
+    var baseURL: URL { get }
+}
+enum CryptoNewsAPI: String {
+    case getNews
+    case getPrices
+}
+extension CryptoNewsAPI: APIBuilder {
+    var baseURL: URL {
+        switch self {
+        case .getNews:
+            return URL(string: FULL_CRYPTO)!
+        case .getPrices:
+            return URL(string: NEW_FULL_COINS)!
+        }
+    }
+    var urlRequest: URLRequest {
```

Comment:
Do we need this var? Because in `Service`, you are not using it, and you are verifying the URL in `fetchNews`. 
Be careful with what you add to your code. Have a look at this code principle [YAGNI](https://betterprogramming.pub/yagni-you-aint-gonna-need-it-f9a178cd8e1)